### PR TITLE
fix # 66781Trill stops after first part of tied note

### PR DESCRIPTION
**Pull request by jimka2001**
if the first of two (or more) notes has a trill (or any other ornament)
   the trill should be played for the entire duration, not just the first note

### CODE REVIEW

**Comment by jimka2001** [comment]
warning, when merging this into master, there may be a merge conflict in the tst_midi.cpp file,
the conflict is easy to resolve.  just that to different pull requests try to insert another test case file
at the same line.  it does not matter which order, but both files need to go in.

**Comment by lasconic** [comment]
I merged the other PR, this one needs to be rebased now.

File: libmscore/rendermidi.cpp [hunk 1]
```diff
@@ -953,6 +953,22 @@ int articulationExcursion(Note *noteL, Note *noteR, int deltastep)
...
 
 //---------------------------------------------------------
+// totalTiedNoteTicks
+//      return the total of the actualTicks of the given note plus
+//      the chain of zero or more notes tied to it to the right.
+//---------------------------------------------------------
+int totalTiedNoteTicks(Note* note)
+      {
+      int total = note->chord()->actualTicks();
+      while ( note->tieFor()) {
```
Comment (lasconic) [inline libmscore/rendermidi.cpp]: That will fail in the case of a note 1 tied to note 2 tied to note 1. Also remove the space in while (note

**Comment by jimka2001** [comment]
what about the fail?  is a tieFor() always forward?  if so then note2 cannot tieFor() to note1.  Right?

**Pull request merged**